src/routes/docs/advanced/platform/+layout.svelte

```diff
@@ -18,6 +18,10 @@
                 {
                     label: 'Shortcuts',
                     href: '/docs/advanced/platform/shortcuts'
+                },
+                {
+                    label: 'Roles',
+                    href: '/docs/advanced/platform/roles'
```

Comment:
Should we also link to this from [here](https://appwrite.io/docs/advanced/platform#:~:text=by%20Appwrite%20APIs.-,Access%20control,-Appwrite%20is%20secure)?

<img width="755" alt="image" src="https://github.com/user-attachments/assets/1c37d86a-6779-4b38-836e-1e16b2fa1b42">

Reply:
No I dont think so. Since the above access control section refers only to client / server access control and not the console 🤔